.eslintrc.js

```diff
@@ -140,5 +170,87 @@ module.exports = {
         'no-use-before-define': 'off',
         '@typescript-eslint/no-use-before-define': ['error'],
         'require-await': ['error'],
+        'no-nested-ternary': 'error', // disallow nested ternary expressions
+        'eol-last': 'error',
+        // add new line before return
     },
+    overrides: [
+        {
+            files: ['**/*.js'],
+            rules: {
+                // JS files are usually configs or scripts where require is OK
+                '@typescript-eslint/no-var-requires': 'off',
+                'import/no-unresolved': 'error',
+            },
+        },
+        {
+            // we are using explicit blacklist because this will enforce new rules in newly created packages
+            files: [
```

Comment:
this blacklist should be updated because lot of packges was added, during last weeks (like connect etc.)